docs/cosmos-hub/gaiacli.md

```diff
@@ -524,37 +524,87 @@ You can also query all of the delegations to a particular validator:
 
 ### Governance
 
-Governance is the process from which users in the Cosmos Hub can come to consensus on software upgrades, parameters of the mainnet or on custom text proposals. This is done through voting on proposals, which will be submitted by `Atom` holders on the mainnet.
+Governance is the process from which users in the Cosmos Hub can come to consensus
+on software upgrades, parameters of the mainnet or signaling mechanisms through
+text proposals. This is done through voting on proposals, which will be submitted
+by `ATOM` holders on the mainnet.
 
 Some considerations about the voting process:
 
-- Voting is done by bonded `Atom` holders on a 1 bonded `Atom` 1 vote basis
+- Voting is done by bonded `ATOM` holders on a 1 bonded `ATOM` 1 vote basis
 - Delegators inherit the vote of their validator if they don't vote
-- **Validators MUST vote on every proposal**. If a validator does not vote on a proposal, they will be **partially slashed**
-- Votes are tallied at the end of the voting period (2 weeks on mainnet). Each address can vote multiple times to update its `Option` value (paying the transaction fee each time), only the last casted vote will count as valid
+- Votes are tallied at the end of the voting period (2 weeks on mainnet) where
+each address can vote multiple times to update its `Option` value (paying the transaction fee each time),
+only the last casted vote will count as valid
 - Voters can choose between options `Yes`, `No`, `NoWithVeto` and `Abstain`
-  At the end of the voting period, a proposal is accepted if `(YesVotes/(YesVotes+NoVotes+NoWithVetoVotes))>1/2` and `(NoWithVetoVotes/(YesVotes+NoVotes+NoWithVetoVotes))<1/3`. It is rejected otherwise
+ - At the end of the voting period, a proposal is accepted if `(YesVotes/(YesVotes+NoVotes+NoWithVetoVotes))>1/2`
```

Comment:
this needs to include the quorum requirement

Reply:
Updated.